tests/test_metadata.py

```diff
@@ -321,6 +338,10 @@ def test_disturl_can_be_checked_out(
     check_output(["osc", "co", disturl], cwd=tmp_path)
 
 
+@pytest.mark.skipif(
+    OS_VERSION == "tumbleweed",
+    reason="no image-type labels on openSUSE containers",
+)
```

Comment:
I think this should work, and would allow you to not repeat the full `skipif` before every function call.
```suggestion
TW_SKIP_MARK = pytest.mark.skipif(
    OS_VERSION == "tumbleweed",
    reason="no image-type labels on openSUSE containers",
)

@TW_SKIP_MARK
```

Reply:
@dcermak the "reason" is always a different string. it is sometimes "supportlabel". so its not easy to remove duplication unless you make a generic "not for opensuse" reason